cirq/experiments/random_quantum_circuit_generation.py

```diff
@@ -217,6 +236,114 @@ def random_rotations_between_two_qubit_circuit(
     return circuit
 
 
+def generate_library_of_2q_circuits(
+    n_library_circuits: int,
+    two_qubit_gate: 'cirq.Gate',
+    *,
+    max_cycle_depth: int = 100,
+    q0: 'cirq.Qid' = devices.LineQubit(0),
+    q1: 'cirq.Qid' = devices.LineQubit(1),
+    random_state: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None,
+) -> List['cirq.Circuit']:
+    """Generate a library of two-qubit Circuits.
+
+    Args:
+        n_library_circuits: The number of circuits to generate.
+        two_qubit_gate: The two qubit gate to use in the circuits.
+        max_cycle_depth: The maximum cycle_depth in the circuits to generate. If you are using XEB,
+            this must be greater than or equal to the maximum value in `cycle_depths`.
+        q0: The first qubit to use when constructing the circuits.
+        q1: The second qubit to use when constructing the circuits
+        random_state: A random state or seed used to deterministically sample the random circuits.
+    """
+    rs = value.parse_random_state(random_state)
+    exponents = np.linspace(0, 7 / 4, 8)
```

Comment:
I would make this an optional argument with defaults set to your value. Is that corect? Or rather the entire single_qubit_gates array, and just fix the defaults to constants?

Reply:
The fully-generic version would be to write your own list comprehension around `random_rotations_between_two_qubit_circuit`. This is a slightly-more abstracted entry point where these things are nailed down:

 - the single qubit rotations
 - the fact that the two qubit gate is constant
 - you want a list of `n_library_circuits` of them

and these things given more opinionated defaults:
 - depth
 - qubit indices (for parallel xeb, they will get re-mapped anyways)

If any of these don't apply it's very straightforward to drop down to `random_rotations_between_two_qubit_circuit`

Reply:
In that case could you at least document  in docstring what kind of signle-qubit rotations are considered and used?

Reply:
added. (also reached out to sergio to see how strong of a theoretical basis there is for this choice. I stole it from the CRNG work)